ShelterModule/Models/Pets/PetUpsertRequest.cs

```diff
@@ -5,18 +5,22 @@ namespace ShelterModule.Models.Pets;
 public sealed class PetUpsertRequest
 {
     [Required]
+    [MaxLength(20)]
```

Comment:
Do we have those requirements in specification? If that's just something that was left from merging an earlier version of shelters branch into this one, we should remove them.